xrviz/dashboard.py

```diff
@@ -81,8 +98,25 @@ def create_plot(self, *args):
             if self.var in list(sel_data.coords):  # When a var(coord) is plotted wrt itself
                 sel_data = sel_data.to_dataset(name=f'{sel_data.name}_')
 
+            c_low_val = self.kwargs['cmap lower limit']
+            c_upp_val = self.kwargs['cmap upper limit']
+
+            # It is better to set initial values as 0.1,0.9 rather than 0,1 i.e. (min, max)
+            # to get a color balanced graph
+            c_lim_lower = float(c_low_val) if c_low_val else sel_data.quantile(0.1)
```

Comment:
I expect `quantile` can take a list argument, to compute both values in one step

Reply:
Now both values are being computed in one step.